docs/monorepo.md

```diff
@@ -51,7 +51,7 @@ The only exception are `devDependencies` which _must be declared in the wp-calyp
 	"module": "dist/esm/index.js",
 	"sideEffects": false,
 	"keywords": [ "wordpress" ],
-	"author": "Your Name <you@example.com> (https://yoursite.wordpress.com/)",
+	"author": "Automattic Inc.",
```

Comment:
I think this is better so as not to promote too tight "code ownership".

I can ping individuals who's names I've switched to the company name in this PR once it's in a mergeable state.